go.mod

```diff
@@ -38,7 +38,7 @@ require (
 	github.com/rs/cors v1.11.0
 	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
 	github.com/seehuhn/mt19937 v1.0.0
-	github.com/spacemeshos/api/release/go v1.42.0
+	github.com/spacemeshos/api/release/go v1.42.1-0.20240530113900-b81297d5714c
```

Comment:
The PR in the api repo needs to be merged first I assume?

Reply:
Yes, I planned to do so after I address all the other comments here